jobs/release-microk8s.yaml

```diff
@@ -33,26 +49,9 @@
             Choose a specific architecture and track to release against
           choices:
             - all
-            - amd64/beta
-            - arm64/beta
-            - amd64/stable
-            - arm64/stable
-            - amd64/prerelease
-            - arm64/prerelease
```

Comment:
i'm not **SURE** but i think running 'all' previously would have tried to build all 3 tracks on both archs.  This has been scaled back since there are two jobs (one per arch) which will build all 3 tracks within that arch